values.yaml

```diff
@@ -23,6 +23,7 @@ securityContext:
 # Most of these map to environment variables. See Minecraft for details:
 # https://hub.docker.com/r/itzg/minecraft-server/
 livenessProbe:
+  enable: false
```

Comment:
change these to `enabled`. past tense is more appropriate for declarative deployments.

Reply:
okei